RealmSwift/Tests/RealmTests.swift

```diff
@@ -489,11 +493,13 @@ class RealmTests: TestCase {
         XCTAssertFalse(realm == testRealm)
 
         let fired = expectationWithDescription("fired")
-        dispatch_async(dispatch_queue_create("background", nil)) {
+        let queue = dispatch_queue_create("background", nil)
+        dispatch_async(queue) {
             let otherThreadRealm = Realm()
             XCTAssertFalse(realm == otherThreadRealm)
             fired.fulfill()
         }
         waitForExpectationsWithTimeout(2, handler: nil)
```

Comment:
That's not a thing that needs to be done. The code was previously just using the expectation to wait for the background block to finish.